helm/ffc-demo-claim-service/values.yaml

```diff
@@ -9,6 +9,11 @@ labels: {}
 appInsights:
   key:
 
+azureIdentity:
+  name: ffc-demo-claim-service-identity
```

Comment:
You don't need this name, it's put in automatically by the template

Reply:
Changes made to remove queue specific usernames & passwords across all files including the `README`.